src/ReachSets/ContinuousPost/BFFPS19/reach_blocks.jl

```diff
@@ -1,4 +1,15 @@
 #helper functions
+@inline function termination_X(k, set, t, X_store_d, blocks, diff_blocks, block_options, termination)
+    terminate, skip, reach_set_intersected = termination(k, set, t)
+    X_store = getX_store(reach_set_intersected, block_options, X_store_d, blocks, diff_blocks)
+    return terminate, skip, X_store
+end
+
+@inline function getX_store(X_store, block_options, X_store_d, blocks, diff_blocks)
```

Comment:
Thanks for outsourcing. May I suggest to sort the arguments such that `X_store` and `X_store_d` are next to each other?